common/BUILD.gn

```diff
@@ -1,6 +1,18 @@
 import("//build/util/branding.gni")
 import("//extensions/buildflags/buildflags.gni")
 
+declare_args() {
```

Comment:
This needs to go in `components/component_updater/BUILD.gn`. Components https://github.com/brave/brave-core/pull/5854/files#diff-5fd6956fd190ef97f5703d18f6c1fa5d cannot depend on `//brave/common`

Reply:
@bridiver - Moved to `components/brave_component_updater/browser` - let me know if that's good - https://github.com/brave/brave-core/pull/5854/files#diff-deb253279892c75adef317ec6e816ce8